src/components/HomeLandingPage/HomeLandingPage.tsx

```diff
@@ -28,24 +29,59 @@ const HomeLandingPage = () => {
   useAllEvents();
 
   // Fetch events from cache
-  const { data } = useQuery(GET_ALL_EVENTS);
+  const { data, refetch } = useQuery(FETCH_ALL_EVENTS);
+  const [editEvent] = useMutation(EDIT_EVENT);
+  const [deleteEvent] = useMutation(DELETE_EVENT);
   const events: Array<Event> = data ? data.events : [];
 
+  const handleArchiveEvent = async (event: Event) => {
+    await editEvent({
+      variables: {
+        id: event.id,
+        name: event.name,
+        eventDate: event.eventDate,
+        createdBy: event.createdBy.id,
+        isActive: false,
+      },
+    });
+    // Force re-render
+    refetch();
```

Comment:
See below, but including the id of the event in the `editEvent` mutation didn't update the cache automatically. I included the refetches so that the event cards update on the client side without the user refreshing their browser